src/index.css

```diff
@@ -52,6 +52,20 @@ body {
   line-height: 28px;
 }
 
+section {
```

Comment:
I love how you are adding semantic html as you find opportunities to do so! This is a good blog post I'm meaning to add to the Knowledge Base, I'm actually thinking of making a separate HTML and separate CSS section away from 'React' and 'JavaScript' and 'UX Design' - I would like them to be a bit more separate. https://markodenic.com/html-tips/